app/code/community/Metrilo/Analytics/Model/Observer.php

```diff
@@ -172,7 +172,7 @@ public function removeFromCart(Varien_Event_Observer $observer)
         $product = $item->getProduct();
 
         $data = array(
-            'id' => $product->getId()
+            'id' => $product->getSku() ? $product->getSku() : $product->getId()
```

Comment:
Would you be willing to find what is not okay with this line?
[hint](https://github.com/Metrilo/metrilo-magento2-integration/pull/73/files#diff-3239ea2def5b5dfe803295a77f337128R18-R20), [hint](https://github.com/Metrilo/metrilo-magento2-integration/pull/74/files#diff-928e1864f3b453f8240a57bcc0d7501dR73-R74)

Reply:
Fixed.